tutorial/10_key_features/002_configurations.py

```diff
@@ -69,22 +69,25 @@ def objective(trial):
 
 ###################################################################################################
 # - Loops:
-import torch
-import torch.nn as nn
-
-
-def create_model(trial, in_size):
-    n_layers = trial.suggest_int("n_layers", 1, 3)
-
-    layers = []
-    for i in range(n_layers):
-        n_units = trial.suggest_int("n_units_l{}".format(i), 4, 128, log=True)
-        layers.append(nn.Linear(in_size, n_units))
-        layers.append(nn.ReLU())
-        in_size = n_units
-    layers.append(nn.Linear(in_size, 10))
-
-    return nn.Sequential(*layers)
+#
+# .. code-block:: python
+#
+#     import torch
+#     import torch.nn as nn
+#
+#
+#     def create_model(trial, in_size):
+#         n_layers = trial.suggest_int("n_layers", 1, 3)
+#
+#         layers = []
+#         for i in range(n_layers):
+#             n_units = trial.suggest_int("n_units_l{}".format(i), 4, 128, log=True)
+#             layers.append(nn.Linear(in_size, n_units))
+#             layers.append(nn.ReLU())
+#             in_size = n_units
+#         layers.append(nn.Linear(in_size, 10))
+#
+#         return nn.Sequential(*layers)
```

Comment:
Let's use sklearn here as well https://scikit-learn.org/stable/modules/generated/sklearn.neural_network.MLPClassifier.html#sklearn-neural-network-mlpclassifier. It'll be less confusing to the reader.

Reply:
Hm, how come? In my understanding, the code is rather simple and self documented. Something along the following.

```python
import sklearn.neural_network import MLPClassifier

def create_model(trial):
    n_layers = trial.suggest_int("n_layers", 3, 5)

    layers = []
    for i in range(n_layers - 2):
        n_units = trial.suggest_int("n_units_l{}".format(i), 4, 128, log=True)  # Irrelevant but I'm not sure if this should be log transformed. 
        layers.append(n_units)

    return MLPClassifier(layers)
```

Reply:
I don't think sklearn MLPClassifier is so common enough to replace torch code.

Reply:
In my understanding, pros/cons for PyTorch are as follows:

* pros
  * We can show a neural net example by using one of the most common library.
* cons
  * We can not detect whether the above code is broken or not by CI

Reply:
Then the pros/cons sklearn are we are free from the ignorance of broken code by using not so popular feature of popular sklearn, right?

Reply:
Well, the example is quite trivial and the main purpose is to demonstrate the usage of Optuna's API, so I think sklearn would do as well. Note that I'm also taking into account that sklearn is used in the previous example. On the other hand, this PR is a temporary fix so it's not a strong opinion. We have to deal with the runtime of the tutorial builds in a more sustainable matter (e.g. introducing new recipes would still quickly exceed the Read the Docs time limit). So, I'm fine either way.